### ISSUE TITLE
.overlay is display:none on a popular bank (ng app)

### ISSUE THREAD

**Issue by operatino** [oblador/hush#31]
I've been debugging another issue I've noticed, which have happened with me while using a bank web app built with Angular. Since it's behind auth, I can't really share the direct link, but here's what I found blocking the important modal, and it seems really generic as `.overlay` class is a very common name for functionally important elements...

See screenshot from dev tools:

![image](https://d.pr/i/XkqkO7+)

**Comment by oblador** [comment]
Hey, `. overlay` is blocked on four specific sites, but not generally. Would you mind sharing the domain?

**Comment by operatino** [comment]
It's https://www.ing.nl, but as I mentioned the issue appeared in the password protected page. I've compares computed styles of the browser window with plugin enabled, and without, and the style shown on the screenshot been the only difference.
